test/models/response_set_test.rb

```diff
@@ -31,44 +70,49 @@ class ResponseSetTest < ActiveSupport::TestCase
   test 'assign_new_oids' do
     prefix = ResponseSet.oid_prefix
 
-    rs1 = ResponseSet.new
-    assert rs1.save
-    assert_equal 1, rs1.version
-    assert_equal "RS-#{rs1.id}", rs1.version_independent_id
-    assert_equal "#{prefix}.#{rs1.id}", rs1.oid
-
-    rs2 = ResponseSet.new
-    rs2.oid = "#{prefix}.#{rs1.id}"
-    assert_not rs2.valid?
-    rs2.oid = "#{prefix}.#{rs1.id + 1}"
-    assert rs2.valid?
-    assert rs2.save
-
-    rs3 = ResponseSet.new
-    rs3.oid = "#{prefix}.#{rs1.id + 3}"
-    assert rs3.save
+    rs = ResponseSet.new(name: 'Test', created_by: @user)
+    rs.id = 3
```

Comment:
No. There was a conflict when I rebased. I guess I didn't get the full intent of you changes while resolving the conflict.